tsconfig.json

```diff
@@ -1,17 +1,19 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "rootDir": "./src",
-    "outDir": "./lib/",
+    "outDir": "./build/",
     "sourceMap": true,
     "types": ["node"],
     "target": "es6",
     "noImplicitAny": true,
     "strictNullChecks": true
   },
+  "include": [
+    "src/"
```

Comment:
It seems to be working both on this module and `resin-bundle-resolve`, e.g. it definitely compiles in all files etc.